<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>804</width>
    <height>592</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/logo/resource/SerialPortYmodem.ico</normaloff>:/logo/resource/SerialPortYmodem.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="aic">
       <attribute name="title">
        <string>爱科微</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="contextMenuPolicy">
           <enum>Qt::ContextMenuPolicy::DefaultContextMenu</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">QGroupBox{
    border-width:1px;
    border-style:solid;
    #border-radius: 5px;
    border-color:gray;
    margin-top:0.5ex;
}</string>
          </property>
          <property name="title">
           <string>串口配置</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>端口：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comPort">
             <property name="font">
              <font>
               <hintingpreference>PreferDefaultHinting</hintingpreference>
              </font>
             </property>
             <property name="layoutDirection">
              <enum>Qt::LayoutDirection::LeftToRight</enum>
             </property>
             <property name="maxVisibleItems">
              <number>12</number>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Orientation::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>波特率：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comBaudRate">
             <property name="editable">
              <bool>true</bool>
             </property>
             <property name="currentText">
              <string>921600</string>
             </property>
             <property name="currentIndex">
              <number>0</number>
             </property>
             <property name="maxVisibleItems">
              <number>12</number>
             </property>
             <item>
              <property name="text">
               <string>921600</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>460800</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>230400</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>115200</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>57600</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>38400</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>19200</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>9600</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>4800</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>2400</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>1200</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Orientation::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="RefreshButton">
             <property name="text">
              <string>刷新</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="comButton">
             <property name="text">
              <string>打开</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_4">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="title">
           <string>日志打印</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QTextEdit" name="Uart_Output">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_9">
             <item>
              <widget class="QLabel" name="label_send">
               <property name="text">
                <string>Send: </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="LineEdit_send"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>Find：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="LineEdit_Search"/>
             </item>
             <item>
              <widget class="QPushButton" name="Button_Find_Previous">
               <property name="text">
                <string>Previous</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="Button_Find_Next">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>Next</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="Button_Clean_Uart">
               <property name="text">
                <string>Clean</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="title">
           <string>文件发送</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>文件路径：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="transmitPath">
               <property name="readOnly">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="transmitBrowse">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>浏览...</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="transmitButton">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>发送</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="Button_semi_auto_load">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>半自动下载</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="Button_fully_auto_load">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>全自动下载</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>传输进度：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QProgressBar" name="transmitProgress">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="contextMenuPolicy">
                <enum>Qt::ContextMenuPolicy::DefaultContextMenu</enum>
               </property>
               <property name="styleSheet">
                <string notr="true">QProgressBar::chunk
{
border-radius:11px;
background:qlineargradient(spread:pad,x1:0,y1:0,x2:1,y2:0,stop:0 #01FAFF,stop:1  #26B4FF);
}
QProgressBar#progressBar
{
height:22px;
text-align:center;
font-size:14px;
color:white;
border-radius:11px;
background: #1D5573 ;
}</string>
               </property>
               <property name="value">
                <number>0</number>
               </property>
               <property name="alignment">
                <set>Qt::AlignmentFlag::AlignCenter</set>
               </property>
               <property name="orientation">
                <enum>Qt::Orientation::Horizontal</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="title">
           <string>文件接收</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>文件路径：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="receivePath">
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="receiveBrowse">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>浏览...</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="receiveButton">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>接收</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>传输进度：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QProgressBar" name="receiveProgress">
               <property name="styleSheet">
                <string notr="true">QProgressBar::chunk
{
border-radius:11px;
background:qlineargradient(spread:pad,x1:0,y1:0,x2:1,y2:0,stop:0 #01FAFF,stop:1  #26B4FF);
}
QProgressBar#progressBar
{
height:22px;
text-align:center;
font-size:14px;
color:white;
border-radius:11px;
background: #1D5573 ;
}</string>
               </property>
               <property name="value">
                <number>0</number>
               </property>
               <property name="alignment">
                <set>Qt::AlignmentFlag::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="apus">
       <attribute name="title">
        <string>国芯微</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_9">
        <item>
         <widget class="QGroupBox" name="groupBox_7">
          <property name="title">
           <string>串口配置</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_13">
           <item>
            <widget class="QLabel" name="label_com">
             <property name="text">
              <string>端口：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comPort_apus">
             <property name="maxVisibleItems">
              <number>12</number>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Orientation::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="label_bandrate">
             <property name="text">
              <string>波特率：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comBaudRate_apus">
             <property name="editable">
              <bool>true</bool>
             </property>
             <property name="currentText">
              <string>921600</string>
             </property>
             <property name="currentIndex">
              <number>0</number>
             </property>
             <property name="maxVisibleItems">
              <number>12</number>
             </property>
             <item>
              <property name="text">
               <string>921600</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>460800</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>230400</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>115200</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>57600</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>38400</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>19200</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>9600</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>4800</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>2400</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>1200</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Orientation::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="RefreshButton_apus">
             <property name="text">
              <string>刷新</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="comButton_apus">
             <property name="text">
              <string>打开</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_6">
          <property name="title">
           <string>日志打印</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <item>
            <widget class="QTextEdit" name="Uart_Output_apus">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_10">
             <item>
              <widget class="QLabel" name="label_send_apus">
               <property name="text">
                <string>Send: </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="LineEdit_send_apus"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_12">
             <item>
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>Find：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="LineEdit_Search_apus"/>
             </item>
             <item>
              <widget class="QPushButton" name="Button_Find_Previous_apus">
               <property name="text">
                <string>Previous</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="Button_Find_Next_apus">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>Next</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="Button_Clean_Uart_apus">
               <property name="text">
                <string>Clean</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_8">
          <property name="title">
           <string>烧录配置</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_10">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_15">
             <item>
              <widget class="QLabel" name="apus_bin">
               <property name="text">
                <string>固件路径：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="binPath_apus"/>
             </item>
             <item>
              <widget class="QPushButton" name="transmitBrowse_apus">
               <property name="text">
                <string>浏览...</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_14">
             <item>
              <widget class="QLabel" name="label_flash">
               <property name="text">
                <string>烧录波特率：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBox_flash_apus">
               <item>
                <property name="text">
                 <string>115200</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>1000000</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>1152000</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>2000000</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_5">
               <property name="orientation">
                <enum>Qt::Orientation::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="Button_flash_apus">
               <property name="text">
                <string>一键烧录</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="debug">
       <attribute name="title">
        <string>调试</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QGroupBox" name="groupBox_5">
          <property name="title">
           <string>软件日志输出</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QTextEdit" name="LogOutput"/>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_11">
             <item>
              <widget class="QPushButton" name="Button_Enter_Boot">
               <property name="text">
                <string>Enter Boot</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="Button_Set_Boot">
               <property name="text">
                <string>Set Boot</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="Button_mcu_cun">
               <property name="text">
                <string>Exit Boot and run</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_8">
             <item>
              <widget class="QPushButton" name="pushButton_function_1">
               <property name="text">
                <string>Function 1</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="ButtonLogClean">
               <property name="text">
                <string>Clean</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
